src/guest_memory.rs

```diff
@@ -386,6 +386,21 @@ impl<T: GuestMemory> Bytes<GuestAddress> for T {
         )
     }
 
+    /// # Examples
```

Comment:
Not related to your changes but can you fix the broken link (`GuestMemroy`) at line 24? (`[GuestMemory](trait.GuestMemroy.html)`)?